### ISSUE TITLE
ERROR: The API Key you have entered does not have access to this site.

### ISSUE THREAD

**Issue by tinusg** [sixfiveinc/fathom-analytics-conversions#1]
In class-fathom-analytics-conversions-admin.php on line 142:

```
if ( $_site_id !== $r_site_id || ( $r_site_name !== $site_name && $r_site_name !== $no_w_s_name ) ) {
    $result['error'] = __( 'ERROR: The API Key you have entered does not have access to this site.', 'fathom-analytics-conversions' );
}
```

It checks for both site_id and site_name, which is undesirable, because the site name in Fathom is not always a domain name (I'd argue in most cases it's not) and it also has a limit of 24 characters. A lot of domain names have more than 24 characters.

I think the extra check for site name can be removed, as site_id is already a unique identifier.

**Comment by 65** [comment]
Good call - thanks for the pick up. We'll get that sorted for the next release.

**Comment by 65** [comment]
Fixed in v1.0.7 please let us know how that goes!
